src/id3_variants_training/ID3_Class.py

```diff
@@ -1,4 +1,5 @@
 import math
+import json
```

Comment:
`import json` seems to be an unused import.

Reply:
Good catch!